### ISSUE TITLE
Padding in Apply theme button on changing theme in simulator

### ISSUE THREAD

**Issue by h2o-Noob** [CircuitVerse/CircuitVerse#2652]
**Describe the bug**
Apply theme button has no internal padding.

**To Reproduce**
1. Go to 'circuitverse.org'
2. Click on 'simulator'
3. Click on 'tools'
4. Click on 'themes'
5. See error

**Expected behavior**
Apply themes button should have a bit of internal padding to make the text more natural and readable.

**Screenshots**
![cv apply btn_LI](https://user-images.githubusercontent.com/85777684/143591781-17c15797-a34d-4278-a5c8-166877544cfe.jpg)

**Device Information [optional]:**

**Additional context**

**Are you working on this issue? (Yes/No)**
Yes, I am working on this issue.

**Comment by vedant-jain03** [comment]
@h2o-Noob This enhancement is also applicable for all the sub-menu popups under the tools menu.
If @nitin10s @tachyons @Shivansh2407  agrees with this you can also add primary-button class in others popups as well.

**Comment by h2o-Noob** [comment]
@vedant-jain03 Yes, I will apply it to all other sub-menu popups as well if I get approval.

**Comment by ayan-biswas0412** [comment]
I don't think it is a bug, rather it is an design enhancement issue.
Looking for your comments @nitin10s

**Comment by tachyons** [comment]
Looks like a bug to me, button text should have padding

**Comment by h2o-Noob** [comment]
> I don't think it is a bug, rather it is an design enhancement issue.
> Looking for your comments @nitin10s

I actually looked at other previous created issues and it seemed like design bugs/flaws are also counted as bug so i labelled it as bug.

**Comment by nitin10s** [comment]
Yes, it's a bug, and we need to change it everywhere. I am working on a set of buttons we should be using and post that we can have PRs making those changes everywhere.

**Comment by Shilpashreemr** [comment]
Can I work on this issue?

**Comment by h2o-Noob** [comment]
> Can I work on this issue?

no sorry, I'm already working on it

**Comment by rohit-raje-786** [comment]
@nitin10s  @tachyons  @ayan-biswas0412 @vedant-jain03 I have resolved the bug🐞 could you please review it.
